physics/PBL/SATMEDMF/satmedmfvdifq.F

```diff
@@ -7,8 +7,6 @@ module satmedmfvdifq
       use mfpbltq_mod
       use tridi_mod
       use mfscuq_mod
-      !PCC_CANOPY
-      use canopy_utils_mod
```

Comment:
@iri01   We still need to use the canopy_utils_mod though for our integrated diffusivity functions. I think you need to revert and put this back.